components/Card.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Card = ({
+  action,
+  actionLabel,
+  address,
+  addressLabel,
+  backgroundImageSource,
+  children,
+  text,
```

Comment:
Perhaps this would be better named as `description` as the use case appears to be to describe the cause?